modules/apps/change-tracking/change-tracking-service/src/main/resources/resource-actions/default.xml

```diff
@@ -2,6 +2,25 @@
 <!DOCTYPE resource-action-mapping PUBLIC "-//Liferay//DTD Resource Action Mapping 7.4.0//EN" "http://www.liferay.com/dtd/liferay-resource-action-mapping_7_4_0.dtd">
 
 <resource-action-mapping>
+	<portlet-resource>
```

Comment:
@stsquared99 <portlet-resource> should be defined in ***-web as it is where Portlet class stays. Why you need to move it to ***-service module?